LICENSE

```diff
@@ -50,3 +50,21 @@ LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
 ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
 (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
 SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
```

Comment:
Right now you have the short form of the license in this LICENSE file, and the full text in `sphinx_immaterial/dropdown.py`.  Can you instead do the reverse: put the full text in this LICENSE file, and the short form or original copyright notice in `sphinx_immaterial/dropdown.py`?

I apologize for having so many requirements regarding the license --- but I hope to replace the copy of this theme in tensorstore with this version, and therefore must follow Google's license guidelines.

Reply:
>I apologize for having so many requirements regarding the license

No worries at all. I admit I can be a little arrogant about these things.

Just to be clear, if we get rid of the CSS that extends makes use of the `details`/`summary` tags, then this new directive (& copyright/license issues) are obsolete. Give me the "ok", and I'll refactor the PR. It would make rebasing easier as well.

Reply:
Yes, sounds good to me.

And I didn't pick up on any arrogance, so no worries about that :)